shelley-ma/impl/src/Cardano/Ledger/Mary/Value.hs

```diff
@@ -8,12 +8,13 @@
 {-# LANGUAGE StandaloneDeriving #-}
 {-# LANGUAGE TypeFamilies #-}
 
-module Cardano.Ledger.ShelleyMA.Value
+module Cardano.Ledger.Mary.Value
```

Comment:
we are going with no special wrapper for Values that are allowed to have negative quantities? I am good with that, but am trying to figure out if that is explicitly the plan or we might add a wrapper later?

Reply:
Good point. I've created https://jira.iohk.io/browse/CAD-2158 for this

Reply:
thank you!